import_map.json

```diff
@@ -21,6 +21,6 @@
     "@std/yaml": "jsr:@std/yaml@1.0.5",
     "xml2js": "npm:xml2js@0.6.2",
     "sentry": "https://deno.land/x/sentry@8.32.0/index.mjs",
-    "@djwt": "jsr:@wok/djwt@3.0.2"
+    "jwt": "jsr:@wok/djwt@3.0.2"
```

Comment:
I would argue to keep the "djwt" so the import keep the same "known" name through the app and there is no confusion with a potential library named the same